core/common/src/TimeZone.kt

```diff
@@ -85,18 +86,36 @@ public expect open class TimeZone {
     public fun LocalDateTime.toInstant(): Instant
 }
 
-@Serializable(with = ZoneOffsetSerializer::class)
-public expect class ZoneOffset : TimeZone {
+@Serializable(with = FixedOffsetTimeZoneSerializer::class)
+public expect class FixedOffsetTimeZone : TimeZone {
+    public constructor(utcOffset: UtcOffset)
+    public val utcOffset: UtcOffset
+
+    @Deprecated("Use utcOffset.totalSeconds", ReplaceWith("utcOffset.totalSeconds"))
+    public val totalSeconds: Int
+}
+
+@Deprecated("Use FixedOffsetTimeZone of UtcOffset instead", ReplaceWith("FixedOffsetTimeZone"))
+public typealias ZoneOffset = FixedOffsetTimeZone
+
+@Serializable(with = UtcOffsetSerializer::class)
+public expect class UtcOffset {
     public val totalSeconds: Int
+
+    public companion object {
+        public fun parse(offsetString: String): UtcOffset
```

Comment:
Parameter name, how would it be better to call it?
- offsetString
- offsetId
- offset
- isoString
- just text as in some other `parse` functions

Reply:
Postpone decision, review all such functions together.